extensions/src/ACRE2Arma/dynload/dynloader.hpp

```diff
@@ -140,6 +170,34 @@ namespace acre {
             return true;
         }
 
+        bool callRVArgs(const arguments & args_, std::string & result_string, int & result_code) {
+            std::string extension_file = args_.as_string(0);
+            if (_modules.find(extension_file) == _modules.end()) {
+                LOG(WARNING) << "callRVArgs could not find module [" << extension_file << "]";
+                return false;
+            } else if (!_modules[extension_file].func_rvArgs) {
+                LOG(ERROR) << "module [" << extension_file << "] does not support CallExtensionArgs";
+                return false;
+            }
+
+            result_string = "";
+            result_string.resize(4096);
+
+            std::string function = args_.as_string(1);
+            std::vector<const char*> cstrings;
+            for (int i = 2; i < args_.size(); i++) {
+                cstrings.push_back(args_.as_string(i).c_str());
+            }
+
+            result_code = _modules[extension_file].func_rvArgs((char*)result_string.c_str(), 4096, function.c_str(), cstrings.data(), static_cast<int>(cstrings.size()));
+#ifdef _DEBUG
+            //if (args_.as_string(0) != "ready") {
```

Comment:
this is already inside an ifdef. Either remove the commented code or remove it alltogether